rainyview/src/main/java/me/samlss/view/RainyView.java

```diff
@@ -262,16 +258,14 @@ public void onRefreshed(Component component) {
         w = component.getWidth();
```

Comment:
variable declaration and initialization can be done in the same statement.

Reply:
Done